recipes/crypten/meta.yaml

```diff
@@ -0,0 +1,59 @@
+{% set name = "crypten" %}
+{% set version = "0.4.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: f9d76bc0ba1468cb77261b8d5486ee875266363310353a18aba948429aa24fda
+  - url: https://raw.githubusercontent.com/facebookresearch/CrypTen/main/requirements.txt
+    sha256: 14569c02e2bb5233d9d75a13646a7e9eaee0a1bcc8e7ffe4044ad9a3511d8eca
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - setuptools
+    - pytest-runner
+  host:
+    - pip
+    - python >=3.6
+    - setuptools
+    - pytest-runner
+  run:
+    - future
+    - setuptools
+    - omegaconf >=2.0.6
+    - onnx >=1.7.0
+    - pandas >=1.2.2
+    - python >=3.6
+    - pyyaml >=5.3.1
+    - scipy >=1.6.0
+    - scikit-learn
+    - tensorboard
+    - pytorch >=1.7.0
+    - torchvision >=0.9.1
+    - pytest-runner
```

Comment:
It is very likely that you do not need `pytest-runner` in the `run` section nor the `host` section.

Can you please confirm that you intend on adding it here?

The requirements do not list it as a package.
https://github.com/facebookresearch/CrypTen/blob/main/requirements.txt

Reply:
Actually, the recipe worked only after putting those in: https://github.com/conda-forge/staged-recipes/pull/15720/commits/d021b9ebc21a9fa371b0a5ec8b5f746fb7d22542

Reply:
I think they made a mistake.
https://github.com/facebookresearch/CrypTen/issues/351

I would leave it as a host dep but remove it as a run dep.

include a link to the issue i raised next to the host dep.

Reply:
So it seems that pytest-runner was an old way to have the command `python setup.py test` https://github.com/pypa/setuptools/issues/1684

But this has lost favor recently to users invoking pytest directly.

Reply:
> I would leave it as a host dep but remove it as a run dep.

Done.